src/lib/resize/index.ts

```diff
@@ -74,28 +74,17 @@ const resize = async (
   }
 }
 
-const IMAGE_SIZE_LIMIT = 20 * 1000 * 1000 // 20MB
 const FILE_SIZE_LIMIT = 30 * 1000 * 1000 // 30MB
```

Comment:
確かサーバーだと30*1024*1024の30MiBだった気がするので、こっちもそれに合わせて良いような気がします。
エラーメッセージを30MiBまでですみたいにするかはわかりやすさと正確性どっちをとるかみたいな問題なので、してもしなくてもいいかなって思ってます
```suggestion
const FILE_SIZE_LIMIT = 30 * 1024 * 1024 // 30MB
```

Reply:
:tashikani:
計算方法は修正して、エラーの表示はMBのままにしようと思います